### DISCUSSION TITLE
"select" menu not opening after I marked overclocke instead of autostart

### DISCUSSION THREAD

**Discussion by MestreALMO** [eduardofilo/RG350_adam_image#278] [SimpleMenu topics]
### What happened?

I wanted to autostart the game and i marked overclock option instead.

When i press "select" in the game i pressed to overclock, it goes to the loading screen and then it goes to the last application/game that i opened in the devide, which makes so i can't open the menu to autostart the game I want to.

If the menu was opening i could have uncheck the overclock and mark the autostart, but without the menu opening, i don't really know what to do to fix it.

If there is a way to fix this please tell me, i really want to take the overclock out and get the autostart feature back in this specific game

### Device

RG350

### Version

v2.0

**Comment by eduardofilo** [comment]
Yes, you have to find and delete a file with the same name as the ROM on which you selected the option by mistake in the simplemenu/rom_preferences directory on the external card. You can do this by mounting the card on the computer or with the Commander application from the console itself.

![screenshot001](https://user-images.githubusercontent.com/2923728/179724034-9f05a11a-fc68-4e37-8ac9-239b4c37c848.png)
